.github/workflows/build.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     - name: test
       run: |
         pip install pytest pytest-cov coverage numpydoc codecov
-        pytest -lv --cov-report term-missing libsvmdata --cov=libsvmdata --cov-config .coveragerc
```

Comment:
can you explain to me what this changes? I wrote this line a long time ago and can't remember all the details...

Reply:
I think that the `libsvmdata` argument tells `pytest` to search in the `libsvmdata` directory to locate testing utilities. I removed it to construct a global `test` directory in the repo root folder (`pytest` default testing utility location).

Reply:
If it was not broken before I wouldn't fix it :)

Reply:
Ok I put the `test` folder back under `libsvmdata/` then 👍